assets/assets.gen.go

```diff
@@ -17,6 +17,8 @@
 
 // Code generated by go-bindata. DO NOT EDIT.
 // sources:
+// .DS_Store (6.148kB)
```

Comment:
Remove this file on your local directory, `assets.gen.go` should be intact in this PR